### DISCUSSION TITLE
IPv6 support

### DISCUSSION THREAD

**Discussion by xqdd** [alvr-org/ALVR#1230] [General]
Both my Oculus Quest2 and ALVR server own a WAN(Internet) IPv6 address and they can mutual access through it, however currently ALVR server does not support IPv6 address.
![image](https://user-images.githubusercontent.com/20871761/199749235-99d0fef0-b7ee-4883-a4ba-2772666dc0b0.png)

**Comment by zmerp** [comment]
The user interface accepts only IPv4 but I think you can override it with IPv6 in the session.json file.

**Reply by xqdd** [reply]
tried but not working
![image](https://user-images.githubusercontent.com/20871761/200000362-58dc79ac-6f09-47b3-81e2-9e821f2cdf96.png)

**Comment by xqdd** [comment]
well, use portforwarding is a workaround...
```sh
# remote desktop
netsh interface portproxy add v4tov6 listenport=9943 listenaddress=* connectport=9943 connectaddress=public ipv6 ip or host
netsh interface portproxy add v4tov6 listenport=9944 listenaddress=* connectport=9944 connectaddress=public ipv6 ip or host

# my local windows desktop(connected same router with quest2, or use port forward function in the router)
netsh interface portproxy add v6tov4 listenport=9943 listenaddress=* connectport=9943 connectaddress=quest2 local ipv4 ip
netsh interface portproxy add v6tov4 listenport=9944 listenaddress=* connectport=9944 connectaddress=quest2 local ipv4 ip

# then set manual ip 127.0.0.1 and select tcp connection type on the remote alvr server
```
